tests/monster_test.fbs

```diff
@@ -18,6 +18,9 @@ enum Color:byte (bit_flags) { Red = 0, Green, Blue = 3, }
 
 union Any { Monster, TestSimpleTableWithEnum, MyGame.Example2.Monster }
 
+union AnyUniqueAliases { M: Monster, T: TestSimpleTableWithEnum, M2: MyGame.Example2.Monster }
```

Comment:
So far the testing of aliases was `tests/union_vector`, since it was coupled with other union features not all languages support. Moving it into this schema enables it for all languages, and it seems that it mostly works (since it is just renaming), but we have to make sure. Alternatively these new types could be moved into `union_vector.fbs` instead.

Reply:
This affects Rust and C++ so I'd like to have this in `moster_test.fbs`.  I'll try to fix C# build.

Reply:
Ah interesting.. @rw will probably want to enable this test for Rust.